PrivacyIDEA_Client/PrivacyIDEA.cs

```diff
@@ -298,9 +316,11 @@ public async Task<bool> UserHasToken(string user, string? domain = null, Cancell
         /// Gets an auth token from the privacyIDEA server using the service account.
         /// Afterward, the token is set as the default authentication header for the HttpClient.
         /// </summary>
+        /// <param name="clientIP">optional client IP to forward to privacyIDEA</param>
+        /// <param name="clientUserAgent">optional client user agent from header to forward to the privacyIDEA</param>
         /// <param name="cancellationToken">optional</param>
         /// <returns>true if success, false otherwise</returns>
-        private async Task<bool> GetAuthToken(CancellationToken cancellationToken = default)
+        private async Task<bool> GetAuthToken(, string? clientIP = null, string? clientUserAgent = null, CancellationToken cancellationToken = default)
```

Comment:
this breaks

Reply:
oops:) [b65fd1e](https://github.com/privacyidea/csharp-client/pull/17/commits/b65fd1e7abbf4004a405a097a3901d04ab61c4e3)